test/Libraries/RevitTestServices/RevitTestPathResolver.cs

```diff
@@ -1,18 +1,26 @@
 ﻿using System.Collections.Generic;
 using System.IO;
 using System.Reflection;
-using Dynamo.Interfaces;
+using TestServices;
 
 namespace RevitTestServices
 {
-    public class RevitTestPathResolver : IPathResolver
+    public class RevitTestPathResolver : TestPathResolver
```

Comment:
Bulk of the code below is removed because `TestPathResolver` already implements them.